pkg/sql/parser/sql.y

```diff
@@ -9633,6 +9653,26 @@ enum_val_list:
     $$.val = append($1.enumValueList(), tree.EnumValue($3))
   }
 
+composite_type_list:
+  name simple_typename
```

Comment:
nit: I think the convention is to wrap `name simple_typename` as a separate object as well. But it looks good to me either way. Feel free to ignore.

Reply:
I see a bunch of other places where we do just this so I'll leave it